doc/conf.py

```diff
@@ -309,7 +309,122 @@
         ('api/api', 'reference/index'),
         ('subsystems/subsystems', 'reference/index'),
         ('kernel/kernel', 'reference/kernel/index'),
-
+		('arc/arduino_101_sss/doc/board.html', 'boards/arc/arduino_101_sss/doc/index.html'),
```

Comment:
So close... missed the ``arc`` folders at the top here that should be ``boards/arc``